src/bootstrap/src/core/build_steps/suggest.rs

```diff
@@ -37,7 +37,7 @@ pub fn suggest(builder: &Builder<'_>, run: bool) {
         .collect::<Vec<_>>();
 
     if !suggestions.is_empty() {
-        println!("==== SUGGESTIONS ====");
+        eprintln!("==== SUGGESTIONS ====");
```

Comment:
Suggestion (heh): these `println!` in `suggest.rs` probably should remain `println` because this is `./x suggest` output, right?